### ISSUE TITLE
AppImage does not run on Ubuntu Xenial

### ISSUE THREAD

**Issue by probonopd** [nvpopov/qppcad#2]
Does not run due to a missing library.
How was this AppImage generated?

```
qppcad: error while loading shared libraries: libQt5Charts.so.5: cannot open shared object file: No such file or directory
Parent pid 8390, child pid 8395
```

Reference:
https://github.com/AppImage/appimage.github.io/pull/1357

**Comment by nvpopov** [comment]
Thank you for your attention.

This is my first attempt with appimage.
The qppcad-x86_64.AppImage generated with travis ci([bash script](https://github.com/nvpopov/qppcad/blob/master/packaging/appimage/make_appimage.sh))
Travis ci build logs - [link](https://travis-ci.org/nvpopov/qppcad/builds/505564575)

According to logs - [link](https://travis-ci.org/nvpopov/qppcad/builds/505564575#L1552) libQt5Charts.so.5 was copied to AppDir.

**Comment by probonopd** [comment]
For whatever reason, the rpath in the qppcad executable is not correct:

```
./qppcad-x86_64.AppImage --appimage-extract
cd ./squashfs-root
patchelf --print-rpath usr/bin/qppcad 
/opt/qt59/lib:/opt/python/3.6.3/lib # not correct; linuxdeployqt should have patched it to look in $ORIGIN/../lib
```

**Comment by probonopd** [comment]
The build log looks like you have used https://github.com/linuxdeploy/linuxdeploy-plugin-qt. In this case, please ask there - may be a bug in that tool.

Or try with linuxdeployqt:
https://github.com/probonopd/linuxdeployqt#using-linuxdeployqt-with-travis-ci

**Comment by nvpopov** [comment]
Fixed in the aee31c8 commit and tested with subset of Debian-based distros(Debian, Mint, Ubuntu).

**Comment by probonopd** [comment]
Thank you very much, it is working for me now 👍
